constants.js

```diff
@@ -73,8 +79,10 @@ const checksWhitelist = {
       modelCheck,
       prTemplateCheck
     ],
+    [openEventGithubActions]: [claCheckGithubAction],
```

Comment:
Adding this to the normal open event should work right?

Reply:
No see the above [comment](https://github.com/oppia/oppiabot/pull/233#discussion_r607936931)